src/BigQuery/Job.php

```diff
@@ -121,7 +127,8 @@ public function cancel(array $options = [])
     }
```

Comment:
Okay. LGTM from BQ requirements perspective.

Is there a standard for handling long-running operations in PHP that might make sense to adopt instead of differently named methods for each kind of operation? I see there are several Promises implementations when I do a search, so maybe not.